ipn/ipnlocal/serve.go

```diff
@@ -516,6 +517,7 @@ func (b *LocalBackend) addTailscaleIdentityHeaders(r *httputil.ProxyRequest) {
 	}
 	r.Out.Header.Set("Tailscale-User-Login", user.LoginName)
 	r.Out.Header.Set("Tailscale-User-Name", user.DisplayName)
+	r.Out.Header.Set("Tailscale-User-Profile-Pic-URL", user.ProfilePicURL)
```

Comment:
`-Url`, not `-URL`.

Otherwise Go will just rewrite it to `Url` in that `Set` call anyway.